src/main/java/project/cinema/controller/AuthenticationController.java

```diff
@@ -0,0 +1,21 @@
+package project.cinema.controller;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import project.cinema.model.dto.UserRequestDto;
+import project.cinema.security.AuthenticationService;
+
+@RestController
+public class AuthenticationController {
+    private AuthenticationService authenticationService;
```

Comment:
```suggestion
    private final AuthenticationService authenticationService;
```
let's add `final` to all autowired fields

Reply:
fix